tracing-appender/src/lib.rs

```diff
@@ -88,12 +88,12 @@
 //!
 //! The [`non_blocking` module][non_blocking]'s documentation provides more detail on how to use `non_blocking`.
 //!
-//! [non_blocking]: ./non_blocking/index.html
+//! [non_blocking]: mod@non_blocking
 //! [write]: https://doc.rust-lang.org/std/io/trait.Write.html
```

Comment:
can we also apply this treatment to docs links to the standard library? it would be fine to do that in a separate PR, though.

Reply:
definitely (tho prefer separate pr), i'm gonna read up more on how those links work for cross crate references :v